src/coreclr/pal/src/thread/context.cpp

```diff
@@ -1183,13 +1183,15 @@ void CONTEXTFromNativeContext(const native_context_t *native, LPCONTEXT lpContex
             lpContext->Fcsr = fpr->fcsr;
             lpContext->Fcc  = fpr->fcc;
             memcpy(lpContext->F, fpr->regs, sizeof(fpr->regs));
+            lpContext->ContextFlags |= CONTEXT_LSX;
         }
         else if (LASX_CTX_MAGIC == info->magic)
```

Comment:
LSX is an SIMD extension, while LASX is advanced SIMD extension. In RISC-V, the "extensions" are recognized in minipal/cpufeatures, perhaps we should align them in the future to keep the code organized.